eth/gasestimator/gasestimator.go

```diff
@@ -82,12 +83,15 @@ func Estimate(ctx context.Context, call *core.Message, opts *Options, gasCap uin
 				if err != nil {
 					return 0, nil, err
 				}
+			} else {
+				available = alternativeFeeBalance
```

Comment:
Before we completely remove CIP-66 from the code base, I would keep the CIP-66 changes in a functioning state.

Since you only set the `available` to the fee-currency balance for CIP-64 transactions, the CIP-66 would now have native token balance as `available`, which is incorrect.